generator/index.js

```diff
@@ -57,15 +58,25 @@ const categoryOrder = ["SmileysAndPeople", "AnimalsAndNature", "FoodAndDrink", "
 async function copyTargetImages(map, target, shouldOptimize) {
     await fs.emptyDir(`../emoji-${target.package}/src/main/res/drawable-nodpi`);
 
-    const sheet = await fs.readFile(`node_modules/emoji-datasource-${target.dataSource}/img/${target.dataSource}/sheets/64.png`);
-    const optimizedSheet = shouldOptimize ? await imagemin.buffer(sheet, {
-        plugins: [
-            imageminPngquant(),
-            imageminZopfli()
-        ]
-    }) : sheet;
-
-    await fs.writeFile(`../emoji-${target.package}/src/main/res/drawable-nodpi/emoji_${target.package}_sheet.png`, optimizedSheet);
+    const src = `node_modules/emoji-datasource-${target.dataSource}/img/${target.dataSource}/sheets/64.png`;
+    const sheet = await Jimp.read(src);
+    const strips = sheet.bitmap.width / 66 - 1;
+    for (let i = 0; i < strips; i++) {
+        const strip = await new Jimp(66, sheet.bitmap.height);
+        await strip.blit(sheet, 0, 0, i * 66, 0, 66, sheet.bitmap.height);
+        const dest = `../emoji-${target.package}/src/main/res/drawable-nodpi/emoji_${target.package}_sheet_${i}.png`;
+        await new Promise((resolve, reject) => strip.write(dest, (err => err ? reject(err) : resolve())));
```

Comment:
It would be great if this could [write to a buffer](https://github.com/oliver-moran/jimp#writing-to-buffers) instead of creating a file, which would be read again and the get overridden if `shouldOptimize` is set.

Reply:
Good call! Done.